src/libs/utils.js

```diff
@@ -180,27 +180,6 @@ export const getDatasetNames = (datasets) => {
   return datasetNames.join('\n');
 };
 
-export const getDatasets = async (darDetails) => {
-  let datasets;
-  await DataSet.getDarDatasets(darDetails.datasetIds).then((resp) => {
-    datasets = resp;
-  });
-  datasets = datasets.map((dataset) => {
-    return find({'propertyName':'Dataset Name'})(dataset.properties);
-  });
-  datasets = map(prop => prop.propertyValue)(datasets);
-  return datasets;
-};
-
-export const applyTextHover = (e) => {
```

Comment:
Both the applyTextHover and removeTextHover are duplicated in code within the PaginationBar.  I like the idea of removing them from utils because they are really tailored to the specific visualization in the PaginationBar.